scripts/update-dependencies/project.json

```diff
@@ -9,7 +9,7 @@
     "Microsoft.CSharp": "4.0.1-rc2-24027",
     "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24027",
     "Microsoft.DotNet.Cli.Build.Framework": "1.0.0-*",
-    "NuGet.Versioning": "3.5.0-rc-1285",
+    "NuGet.Versioning": "3.5.0-rc-869",
```

Comment:
do not merge this PR until the NuGet version we are pulling in is an official version.